src/libpriv/rpmostree-importer.c

```diff
@@ -662,9 +662,16 @@ compose_filter_cb (OstreeRepo         *repo,
         }
     }
 
+  /* Special case exemptions */
+  if (g_str_has_prefix (path, "/usr/etc/selinux") &&
+      g_str_has_suffix (path, ".LOCK"))
+    {
+      /* These empty lock files cause problems;
+       * https://github.com/projectatomic/rpm-ostree/pull/1002
+       */
+      return OSTREE_REPO_COMMIT_FILTER_SKIP;
+    }
```

Comment:
So, to clarify, we don't actually *need* this anymore now that we have `force_copy_zerosized`, right? (I mean, it's a good idea regardless, just making sure I understand the intent).

Reply:
Yep.